<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateBook</class>
 <widget class="QDialog" name="updateBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Update</string>
  </property>
  <widget class="QLabel" name="updateBookTilte">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Book Update</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ISBN:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="isbnInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yearInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly|Qt::ImhNoPredictiveText</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Publisher:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="publisherInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="genreInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="priceInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qtyInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhFormattedNumbersOnly</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Title: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="titleInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Search">
        <property name="text">
         <string>Search for book</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Update Book</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Search</sender>
   <signal>clicked()</signal>
   <receiver>updateBook</receiver>
   <slot>enablebutton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Search</sender>
   <signal>clicked()</signal>
   <receiver>updateBook</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateButton</sender>
   <signal>clicked()</signal>
   <receiver>updateBook</receiver>
   <slot>updatedBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enablebutton()</slot>
  <slot>search()</slot>
  <slot>updatedBook()</slot>
 </slots>
</ui>
